code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -276,14 +276,13 @@
 
 /obj/item/reagent_containers/hypospray/autoinjector/isotonic
 	name = "isotonic solution autoinjector"
-	desc = "An auto-injector loaded with a single dose of isotonic solution, formulated to quickly recover fluid volume after blood loss or trauma."
+	desc = "An auto-injector loaded with 2 doses of isotonic solution, formulated to quickly recover fluid volume after blood loss or trauma."
 	icon_state = "autoinjector-8"
-	amount_per_transfer_from_this = 25
-	volume = 25
+	amount_per_transfer_from_this = 15
+	volume = 30
 	list_reagents = list(
-		/datum/reagent/iron = 10,
-		/datum/reagent/consumable/nutriment = 10,
-		/datum/reagent/consumable/sugar = 5,
+		/datum/reagent/iron = 15,
+		/datum/reagent/medicine/saline_glucose = 15,
```

Comment:
Do you think marine blood regen rate is too low at the moment? You're increasing it by a third with the 'default' mix.

Reply:
I think isotonics should be purely about regenning blood and not just nutrition, and yes, I think even chugging the fuck out of isotonics still regens blood very slow.